lib/publiccloud/azure.pm

```diff
@@ -173,7 +173,7 @@ sub get_image_definition {
     foreach my $def (@$json_data) {
         my $identifier = $def->{identifier};
         next unless (defined $identifier);
-        if ($identifier->{publisher} =~ /$publisher/i && $identifier->{offer} =~ /$offer/i && $identifier->{sku} =~ /$sku/i) {
+        if ($identifier->{publisher} eq $publisher && $identifier->{offer} eq $offer && $identifier->{sku} eq $sku) {
```

Comment:
but ... why?

Reply:
When I explained that change yesterday to Anton I was not able to find any example.
The expression was also pretty benevolent so I decided to just leave the `record_info`'s for now.